virt/virtual_machines/virt-automating-windows-sysprep.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="virt-automating-windows-sysprep"]
+= Automating Windows installation with sysprep
+include::modules/virt-document-attributes.adoc[]
+:context: virt-automating-windows-sysprep
+
+toc::[]
+
+You can use Microsoft DVD images and `sysprep` to automate Windows installation, setup, and software provisioning for virtual machines.
+
+include::modules/virt-using-windows-dvd-disk-image.adoc[leveloffset=+1]
+include::modules/virt-using-disk-image-install-windows.adoc[leveloffset=+1]
+include::modules/virt-generalizing-windows-sysprep.adoc[leveloffset=+1]
+include::modules/virt-specializing-windows-sysprep.adoc[leveloffset=+1]
+
+[id="additional-resources_virt-automating-windows-sysprep_{context}"]
+== Additional resources
+link:https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation[Microsoft, Sysprep (Generalize) a Windows installation]
```

Comment:
These should probably be a bullet list. Right now, they are rendering as a single line of links. I would also try to expand the text a little so that the user knows what the links point to.